HA CLI Bug - AARCH64

## Describe the problem

The newest build seems to have an issue with at least requesting DNS info.  Returns an error with any syntax.

## Debug information

1. SSH Addon type

- [X] Community
- [ ] Core
- [ ] Custom

2. Version of the CLI

CLI Version:  latest (4.30.0)
Add-on: 
Add-on version: 

3. Version of Home Assistant Core & the Home Assistant Supervisor

Home Assistant Core: Developer Latest
Home Assistant Supervisor: Developer Latest

4. Output from executing the command with debug info

e.g., `ha hardware info` should be run as `ha --log-level debug hardware info`
```txt
FAILED
The arch that is used is aarch64 thus: 
https://github.com/home-assistant/cli/releases/download/4.30.0/ha_aarch64
```

## Expected behavior

It should return something.

## Additional context

Linux local-openziti 6.6.12-linuxkit #1 SMP Thu Feb  8 06:36:34 UTC 2024 aarch64 Linux

root@local-openziti /> ./ha_aarch64 
panic: "ro" shorthand is more than one ASCII character

goroutine 1 [running]:
github.com/spf13/pflag.(*FlagSet).AddFlag(0x4000099a00, 0x4000148820)
        /home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:864 +0x44c
github.com/spf13/pflag.(*FlagSet).VarPF(0x8b9120?, {0x5c70b8, 0x40000cf6a0}, {0x4d20aa, 0x9}, {0x4cf376, 0x2}, {0x4ec673, 0x34})
        /home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:831 +0x15c
github.com/spf13/pflag.(*FlagSet).BoolVarP(0x8985c0?, 0x3d44e8?, {0x4d20aa?, 0x4d09aa?}, {0x4cf376?, 0x2?}, 0x0?, {0x4ec673?, 0x4df6e7?})
        /home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.5/bool.go:55 +0x48
github.com/spf13/pflag.(*FlagSet).BoolP(...)
        /home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.5/bool.go:80
github.com/home-assistant/cli/cmd.addMountFlags(0x8a3800?)
        /home/runner/work/cli/cli/cmd/mounts.go:36 +0x33c
github.com/home-assistant/cli/cmd.init.86()
        /home/runner/work/cli/cli/cmd/mounts_add.go:74 +0x28

--- a/cmd/mounts.go
+++ b/cmd/mounts.go
@@ -33,7 +33,7 @@ func addMountFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP("password", "p", "", "Password to use for authentication (cifs mounts only)")
 	cmd.Flags().StringP("version", "v", "", "Version to use for the mount (cifs mounts only)")
 	cmd.Flags().StringP("path", "a", "", "Path to mount (nfs mounts only)")
-	cmd.Flags().BoolP("read-only", "ro", false, "Is mount read-only (not available for backup mounts)")
+	cmd.Flags().Bool("read-only", false, "Is mount read-only (not available for backup mounts)")
 
 	cmd.Flags().Lookup("read-only").NoOptDefVal = "true"
 	cmd.RegisterFlagCompletionFunc("type", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {

